pkg/agent/datamodel/linux_sig_version.json

```diff
@@ -0,0 +1,4 @@
+{
+  "ostype": "LinuxSIGImageVersion",
```

Comment:
doesn't look like "ostype" is ever used? I might be missing something tho

Reply:
Currently not used but it will be used later to differentiate between different SKUs (while using same helper function)